components/Nav.js

```diff
@@ -1,42 +1,176 @@
 const Nav = ({ children }) => (
   <nav>
     <h1>Cognition Brewery</h1>
-    <ul>
-      {React.Children.map(children, child => (
-        <li>{child}</li>
-      ))}
-    </ul>
+    <div id="menuToggle">
```

Comment:
A good rule of thumb here is "`id` is for links, `className` is for styling" … you could do the same with `className="menuToggle"` and changing all the below from `#menuToggle` to `.menuToggle`